server/prisma/schema.prisma

```diff
@@ -26,9 +26,18 @@ model User {
   createdAt                DateTime                   @default(now())
   googleId                 String?                    @unique
   facebookId               String?                    @unique
+  Token                    Token[]
```

Comment:
Can a user have multiple tokens for each service?

Reply:
updated